examples/drupal_loadtest.rs

```diff
@@ -97,12 +97,12 @@ async fn drupal_loadtest_front_page(client: &mut GooseClient) {
             }
             Err(e) => {
                 eprintln!("failed to parse front page: {}", e);
-                client.set_failure(&GooseMethod::GET, "/");
+                client.set_failure(&mut response.request);
```

Comment:
While I like the long form this feels like an excellent use case for another macro - `!fail(response)`

etc.

Of course it might also hide things.